do Search error fixed

jwalapc opened the pull request:
Finally fixed the dosearch() error

walterbender commented:
Pull Request 101:
(1) What is the problem you are trying to solve?
(2) What is your approach?
(3) How did you test your solution?

When I try to run this branch, Music Blocks does not load. (Chromium 91 on Fedora 34)

jwalapc commented:
1)There was a bug :
![Screenshot from 2021-08-19 12-39-12](https://user-images.githubusercontent.com/59509045/130359774-6939f325-3211-4d2b-a2db-ab088cc1e925.png)
2) dosearch() is used to search for blocks on the palette.
3)I added the source file name activity.js and    $.noConflict();
4) This solution fixes the #2618 issue.
5)Please review it i had fixed. @walterbender  .

walterbender commented:
Before I test, can you please describe how you tested that this works?

jwalapc commented:
Uses JQuery to add autocompleted search suggestions
Before this functionality in the palettes was having bugs,
After I change the code this functionality is fixed.
In this way I tested my solution.
And if I am wrong please help me @walterbender

walterbender commented:
So you actually ran the code and was able to successfully do a search? Which browser/OS?

jwalapc commented:
> So you actually ran the code and was able to successfully do a search? Which browser/OS?


Firefox/ubuntu

walterbender commented:
Maybe you forgot to push something. It still fails to load for me on both FireFox and Chromium.

jwalapc commented:
> Maybe you forgot to push something. It still fails to load for me on both FireFox and Chromium.
If you don't mind can you show me ?
Because I think I had pushed everything
